Instructions/Labs/AZ-204_05_lab_ak.md

```diff
@@ -429,7 +429,7 @@ In this exercise, you created a .NET console application to display a machine’
 
 1.  From the **ContainerCompute** blade, select the container registry that you created earlier in this lab.
 
-1.  From the **Container Registry** blade, select **Update**.
```

Comment:
Right now, we only have two options for the name of the blade:
- The unique name given by the lab user
- Container registry

The pattern from other labs is to use the subtitle when the title is globally unique (like in this example). The "Overview" option is not the name of the blade, but an option in the navigation menu. 

In the pre-req Azure Fundemantals content, students are taught about blades and this could lead to some confusion.

![image](https://user-images.githubusercontent.com/5067401/115719183-da93ee00-a349-11eb-84bb-81e90cbd8181.png)